game/stronghold_crusader_aiv.ksy

```diff
@@ -0,0 +1,104 @@
+meta:
+  id: aiv
+  title: AI Village
+  application:
+    - Stronghold Crusader
+    - Stronghold Crusader Extreme
+    - Village
+  file-extension: aiv
+  license: CC0-1.0
+  endian: le
+doc: |
+  The structure of the Stronghold Crusader AI Village file format is similar to
+  the one of the map file format. It consists of a directory, specifying
+  properties of the following data sections. Some sections are compressed using
+  blast compression, an implementation of which can be found at
+  https://github.com/ladislav-zezula/StormLib/tree/master/src/pklib.
+seq:
+  - id: dir
+    type: dir
+  - id: x_view
+    type: u4
+  - id: y_view
+    type: u4
+  - id: random
+    size: 40016
+  - id: bmap_size
+    type: compr_sec
+  - id: bmap_tile
+    type: compr_sec
+  - id: tmap
+    type: compr_sec
+  - id: gmap
+    size: 10000
```

Comment:
It is not obvious for people that are not familiar with this format what these abbreviations mean. Could you expand them for more clarity?